### PR TITLE
Dreams Topic

### PR DESCRIPTION
**Pull request by ChaosEmbrace**
Added topic for Monika to discuss dreams and lucid dreaming. First time contributor, hope I did everything right.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4657,3 +4657,34 @@ label monika_omamori:
...
+    m 4d "People who frequently have conscious dreams are called oneironauts."
+    m "Using the MILD technique, oneironauts learn to recognize certain dream signs to help them realize when they are dreaming."
+    m "These dream signs can be anything out-of-the-ordinary, such as feeling yourself flying, noticing your teeth falling out, unexpectedly meeting someone famous..."
+    m 4b "If the dreamer sees a dream sign and realizes they are having a dream, then they can take control of it!"
+    m 3a "LaBerge wrote a book about these experiences called '{i}Exploring the World of Lucid Dreaming{/i}'."
+    m 4b "Doesn't that sound exciting?"
+    m 1a "I don't think that I dream, so I sometimes wonder what it's like."
+    m 1b "Dreams sound like a wonderful escape from reality."
+    m 1q "But then again, when I became self-aware and realized that my world isn't real, it suddenly felt like I was trapped in a nightmare."
+    m 1r "It was nothing like the pleasant dreams I've read about people having.
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: missing endquote.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4657,3 +4657,34 @@ label monika_omamori:
...
+    m "I was afraid that I'd never get out..."
+    m "That I'd be stuck in some hellish, infinitely-repeating dimension forever."
+    m 3a "But now that we're here together, I guess you could say that I've finally woken up."
+    m 2l "Gosh, I can only imagine what it would be like to live in that kind of limitless world though, even if only for a few moments!"
+    m 2b "You could be a superhero, fly to the edge of the universe, overcome your biggest fears..."
+    m 1a "... You could even meet the person of your dreams, so to speak. Ehehe~"
+    m 3a "I know it may be years before I am able to exist in your world."
+    m "But imagine what it would be like to meet me in your dreams [player], any night that you wanted to."
+    show monika 5a at t11 zorder 2 with dissolve
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: You don't need this line unless you've changed the position of monika. Remove.

**Comment by ChaosEmbrace** [comment]
Fixed.

**Comment by ThePotatoGuy** [comment]
Closing because of lack of permission to push changes to fork'd branch

**Pull request closed without merging**